cypress/integration/lists_test/enrichment_test.js

```diff
@@ -0,0 +1,99 @@
+        })
+    })
+
+    it("can view and download single enrichment result", function(){
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.get('.enrichment-filter').within(() => {
+                cy.get("select").select("molecular_function");
+            })
+            cy.contains("cell adhesion molecule binding").click();
+        })
+        cy.get('.pagination-label').should("include.text","4 rows");
+        cy.get(".enrichment-category").eq(0).within(() => {
+            cy.get(".icon-download").click();
+        })
+        cy.readFile("cypress/downloads/GO_0050839 Gene Ontology Enrichment.tsv").then(newResult => {
+            cy.readFile("cypress/fixtures/single_enrichment_result.tsv").should("eq",newResult);
+        })
+    })
+    it("can view and download checked enrichment results", function(){
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.get('[type="checkbox"]').eq(1).check();
+            cy.get('[type="checkbox"]').eq(2).check();
+            cy.get('[type="checkbox"]').eq(1).should("be.checked");
+            cy.get('[type="checkbox"]').eq(2).should("be.checked");
+            cy.contains("View").click();
+        })
+        cy.get('.pagination-label').should("include.text","22 rows");
+        cy.get(".enrichment-category").eq(0).within(() => {
+            cy.get(".icon-download").click();
+        })
+        cy.readFile("cypress/downloads/Enrichment Results Gene Ontology Enrichment.tsv").then(newResult => {
+            cy.readFile("cypress/fixtures/checked_enrichment_result.tsv").should("eq",newResult);
+        })
+    })
+
+    it("can filter enrichment results by ontology", function(){
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.intercept("POST","/biotestmine/service/list/enrichment").as("enrichmentLoad");
+            cy.get(".enrichment-filter > div > select").select("molecular_function");
+            cy.wait("@enrichmentLoad");
+            cy.contains("Gene Ontology Enrichment").should("include.text","(1)");
+        })
+    })
+
+    it("can select test correction method and filter enrichment results by text", function(){
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.intercept("POST","/biotestmine/service/list/enrichment").as("enrichmentLoad");
+            cy.contains("Max p-value").parent().find("select").eq(0).select("0.10");
+            cy.get(".correction").find("select").select("Benjamini Hochberg");
+            cy.get(".text-filter > .form-control").type("x");
+            cy.get(".text-filter > .form-control").click().wait(500).clear().type("symbiosis{enter}",{delay:100}); //very flaky
+            cy.wait("@enrichmentLoad");
+            cy.get(".enrichment-item").eq(0).should("include.text","symbiosis");
+            cy.get(".enrichment-p-value").eq(0).should("have.text","1.783148e-7");
+        })
+    })
```

Comment:
This is fine, but can you also add tests which only test one specific filter, for *max p-value* and *test correction*, similar to "can filter enrichment results by ontology" checking that the count has changed (to make this obvious, you can check the count prior to setting the filter, and then after).